pkg/genericapiserver/config.go

```diff
@@ -318,16 +313,13 @@ func (c Config) New() (*GenericAPIServer, error) {
 	}
 
 	if c.RestfulContainer != nil {
-		s.mux = c.RestfulContainer.ServeMux
```

Comment:
So we never use c.RestfulContainer.ServeMux?
Doesnt this break anyone who is setting that and expecting it to be used.

I always assumed that its openshift that is setting ServeMux. Is it not?